### DISCUSSION TITLE
GitHub Student Pack Limits

### DISCUSSION THREAD

**Discussion by audreykoz** [community/community#22145] [New to GitHub]
<p>Hi,<br>
I’m currently a student and meet the qualifications for the Student Pack, but I was wondering if I could use the Student Pack towards internships I work for while I’m a student.<br>
Thanks!</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>Work you do as an intern is absolutely fine (subject to our our <a href="https://help.github.com/en/github/site-policy/github-terms-of-service#c-acceptable-use">Terms of Service</a> of course)</p>
<p>You can <a href="https://education.github.com/pack">apply here</a>!</p>
